Maple2.Model/Metadata/Table/ExpBaseTable.cs

```diff
@@ -0,0 +1,5 @@
+﻿using System.Collections.Generic;
+
+namespace Maple2.Model.Metadata;
+
+public record ExpBaseTable(IReadOnlyDictionary<int, IReadOnlyDictionary<int, long>> Entries) : Table;
```

Comment:
Can we combine this and `NextExpTable.cs` into a single file called `ExpTable.cs`